test/runtests.sh

```diff
@@ -16,4 +16,4 @@ if [ ! -f $ch_path ]; then
     exit 1
 fi
 
-"$test_path/runtests.py" --binary $ch_path Basics/hello.js
+"$test_path/runtests.py" -t $ch_path $test_path/Basics/hello.js
```

Comment:
Removing "--binary" might break it. It may treat `$ch_path` as an input test file. Then `-t` build type takes effect and it'll look for binary `BuildLinux/test/ch`.

The CI should be updated to "build.sh -d" for Debug and "build.sh -t" for Test; and pass -d or -t here.